apps/prairielearn/src/pages/instructorAssessmentQuestionStatistics/instructorAssessmentQuestionStatistics.ts

```diff
@@ -49,21 +51,29 @@ router.get(
     });
```

Comment:
This comment is for lines 39 & 49 (unable to comment on unchanged lines directly). Do we want to make these Zod-validated queries?

Reply:
Good idea, let me make those changes.

Reply:
Actually: I'm going to leave `select_assessment` alone so that it will always behave consistently with the usual `res.locals.assessment` that we get from the `selectAndAuthzAssessment` middleware. I'll change the `assessment_stats_last_updated` query though.

Reply:
Changed in 31220a2.